cmd/manifest/init-v2.go

```diff
@@ -577,7 +517,8 @@ func (mc *ManifestCommand) getManifest(path string) (*model.Manifest, error) {
 		manifest.Build = b
 		d := model.NewDeployInfo()
 		if mc.manifest.Deploy != nil {
-			copy(d.Commands, mc.manifest.Deploy.Commands)
+			d.Commands = mc.manifest.Deploy.Commands
```

Comment:
I changed this `copy` because it didn't work and did not copy the commands, so it would never deploy anything.

I've checked and by changing `mc.manifest.Deploy.Commands` after the assignment to `d.Commands`, it does not affect `d.Commands`.